### PR TITLE
Switches all camel casing to snake case

### PR DESCRIPTION
**Pull request by DucksAlongTheLake**
PEP8 is word of law

### CODE REVIEW

**Comment by DucksAlongTheLake** [comment]
@wheeler-dylan go ahead and throw a review in and merge when you are ready.  I would but this is gonna be straight to master

File: GameItems/gameItem.py [hunk 1]
```diff
@@ -114,90 +117,90 @@ def quickBuild(self):
...
-    itemsList = []
+def load_items_list():
+    items_list = []
     
-    for eachFile in glob("./GameItems/*.gmitm"): #search game items directory for all gmitm files
-        thisItem = gameItem()
-        thisItem.loadItemFromFile(open(eachFile))
-        itemsList.append(thisItem)
+    for each_file in glob("./GameItems/*.gmitm"): #search game items directory for all gmitm files
+        this_item = game_item()
```
Comment (wheeler-dylan) [inline GameItems/gameItem.py]: ERROR: game_item() not defined, needs to match class name.

File: PlayerCharacter/characterCreation.py [hunk 2]
```diff
@@ -120,41 +120,41 @@ def abilitySpinboxUpdate():
...
-    for eachSkill in skills.coreSkills.values():
-        skillBonuses[eachSkill.ID] = 0
+    skill_bonuses = {}
+    for each_skill in skills.core_skills.values():
+        skill_bonuses[each_skill.id] = 0
 
 
     #give instructions for adding points to skill ranks
-    skillBanner = str("Skills:\nYou have " + str(skillPointsLeft) + " "+ 
+    skil_banner = str("Skills:\nYou have " + str(skill_points_left) + " "+
```
Comment (wheeler-dylan) [inline PlayerCharacter/characterCreation.py]: ERROR: skil_banner not defined, needs to be skill_banner

File: sandbox.py [hunk 3]
```diff
@@ -95,32 +95,32 @@
...
         print("\n-------------------------\n")
 
     #tests game item load and print
     elif (command == "find"):       
         print("-------------------------\n")
-        ironSword = gameItem.gameItem()
-        ironSword.loadItemFromFile(open("./GameItems/ironsword.gmitm"))
-        ironSword.printItem()
-        player1.collectItem(ironSword)
+        ironSword = gameItem.game_item()
```
Comment (wheeler-dylan) [inline sandbox.py]: ERROR: game_item() not defined, needs to match class name.

File: sandbox.py [hunk 4]
```diff
@@ -95,32 +95,32 @@
...
+        ironSword.load_item_from_file(open("./GameItems/ironsword.gmitm"))
+        ironSword.print_item()
+        player1.collect_item(ironSword)
         print("\n")
-        journal = gameItem.gameItem()
```
Comment (wheeler-dylan) [inline sandbox.py]: ERROR: game_item() not defined, needs to match class name.

File: sandbox.py [hunk 5]
```diff
@@ -95,32 +95,32 @@
...
-        journal.loadItemFromFile(open("./GameItems/journal.gmitm"))
-        journal.printItem()
-        player1.collectItem(journal)
+        journal = gameItem.game_item()
```
Comment (wheeler-dylan) [inline sandbox.py]: ERROR: game_item() not defined, needs to match class name.

File: sandbox.py [hunk 6]
```diff
@@ -95,32 +95,32 @@
...
         print("\n-------------------------\n")
 
     #tests GUI item creation
     elif (command == "craft"):      
         print("-------------------------\n")
-        newItem = gameItem.gameItem()
-        gameItem.gameItemCreation(newItem)
-        newItem.printItem()
-        player1.collectItem(newItem) #add to inventory
+        new_item = gameItem.game_item()
```
Comment (wheeler-dylan) [inline sandbox.py]: ERROR: game_item() not defined, needs to match class name.

**Pull request merged**